Sources/Alchemy/SQL/QueryBuilder/Query.swift

```diff
@@ -342,6 +604,11 @@ public class Query: Sequelizable {
         }
     }
 
+    /// Perform an update on all data matching the query in the builder with the values provided.
+    ///
```

Comment:
Good spot for an example so the update syntax is clear
```suggestion
///
/// Usage:
/// ```
/// DB.query()
///     .table("users")
///     .where("id" == 10)
///     .update(values: [
///         "first_name": "Ashley"
///     ])
/// ```
///
```